<launch>
  <arg name="calib_radar" default="false"/>
  
  
  <rosparam file="$(find radar_ros)/cfg/params_candrive.yaml" command="load" />
  <node pkg="radar_ros" type="radar_ros_node" name="radar_candrive" output="screen">
  </node>
  <node pkg="radar_ros" type="radar_filter_node" name="radar_filter" output="screen">
  </node>
   <node pkg="radar_ros" type="radar_combine_node" name="radar_combine" output="screen">
  </node>
  
  <group if="$(arg calib_radar)">
  <node pkg="radar_ros" type="dynamic_calib_node" name="visualize_calib" output="screen">
  </node>
  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_visualize" output="screen">
  </node>
  </group>

</launch>
